src/VisualStudio/Core/Impl/CodeModel/RootCodeModel.cs

```diff
@@ -39,8 +41,8 @@ private RootCodeModel(CodeModelState state, EnvDTE.Project parent, ProjectId pro
         private Project GetProject()
             => Workspace.CurrentSolution.GetProject(_projectId);
 
-        private Compilation GetCompilation()
-            => GetProject().GetCompilationAsync().Result;
```

Comment:
so old we're still calling .Result, not even GetAwaiter().GetResult();

Reply:
Yep, at least 10 years old from the blame.